modules/yoastSeo/lib/Requirement/ReadabilityAnalysis.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @package     PublishPress\Checklists
+ * @author      PublishPress <help@publishpress.com>
+ * @copyright   Copyright (C) 2018 PublishPress. All rights reserved.
+ * @license     GPLv2 or later
+ * @since       1.0.0
+ */
+
+namespace PublishPress\Checklists\Yoastseo\Requirement;
+
+
+use PublishPress\Checklists\Core\Requirement\Base_simple;
+use stdClass;
+
+class ReadabilityAnalysis extends Base_simple
```

Comment:
Please, use a consistent pattern for the class name (and file name). The other requirements are following the snake case with capital letters. Sorry, I know we have different patterns for other files, but we are working to define a standard pattern for our plugins and will refactor the code in a near future. But it would be good to keep consistency among similar classes for now. Something like "Readability_Analysis" would work better.